_posts/en/newsletters/2020-09-30-newsletter.md

```diff
@@ -0,0 +1,133 @@
+---
+title: 'Bitcoin Optech Newsletter #117'
+permalink: /en/newsletters/2020/09/30/
+name: 2020-09-30-newsletter
+slug: 2020-09-30-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter describes a compiler bug that casts doubt on the
+safety of secure systems and includes our regular sections with popular
+questions and answers from the Bitcoin StackExchange, announcements of
+releases and release candidates, and summaries of notable changes to
+popular Bitcoin infrastructure software.
+
+## Action items
+
+*None this week.*
+
+## News
+
+- **Discussion about compiler bugs:** a [test][oconnor test] written
+  last week by Russell O'Connor for libsecp256k1 failed due to a
+  [bug][gcc bug] in the GNU C Compiler's (GCC's) built-in version of the
+  standard `memcmp` function.  This function takes two regions of memory
+  and compares them, returning whether the first region would be less
+  than, equal to, or greater than the second region if they were both
+  integer values.  This is a commonly used low-level function.
```

Comment:
The last part of the sentence `if they were both integer values.` really drags.

Maybe `This function takes two regions of memory returning whether the first region would be less
  than, equal to, or greater than the second regionand compares these integer values, returning whether the first region would be less  than, equal to, or greater than the second region.`

Not sure if this is still technically correct.